app/src/main/java/com/example/mbanking/pages/AllTransactions.kt

```diff
@@ -36,14 +36,30 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.example.mbanking.R
+import com.example.mbanking.details.DateBottomSheet
 import com.example.mbanking.details.TransactionCard
+import com.example.mbanking.util.sortByDate
+
+
+var newStartDate = "01.01.1970"
+var newEndDate = "30.01.3000"
 
 @Composable
 fun AllTransactions(
     modifier: Modifier = Modifier,
     onClickBackButton: () -> Unit,
     onClickTransactionButton: () -> Unit
 ) {
+    var startDate by remember { mutableStateOf("") }
+    var endDate by remember { mutableStateOf("") }
+    startDate = newStartDate
+    endDate = newEndDate
```

Comment:
you can init vars in single line of code: var startDate by remember { mutableStateOf(newStartDate) } for example

Reply:
if i will do like this my lazylist will not update

Reply:
if you will make your work done for all comments (use lambdas to ser start and end dates from filter screen) this variables didn't make sense anymore. Or they might convert to const values and you could use it as init values. Deal with other comment I wrote for other parts of your code, please

Reply:
done